README.md

```diff
@@ -185,6 +185,37 @@ eval $(ssh-agent)
 ssh-add
 ```
 
+#### The Things Network Requirements
+
+To be able to run the automatic tests in spec [11-lorawan](11-lorawan/test_spec11.py) a valid TTN
+account is required, and an application needs to be created with a device configure
+to do `OTAA` and another configured to do `ABP`.
+
+- [create TTN account](https://www.thethingsnetwork.org/docs/devices/node/quick-start/#create-an-account)
+- [add an application](https://www.thethingsnetwork.org/docs/applications/add/), take note of:
+  - Application ID as `TTN_APP_ID`
+- [register two devices](https://www.thethingsnetwork.org/docs/devices/node/quick-start/#register-your-device)
+  one for `ABP` and a second for `OTAA`
+- personalize one [device for ABP](https://www.thethingsnetwork.org/docs/devices/registration/#personalize-device-for-abp),
```

Comment:
Can we make all the links (the once below as  well) references please? It makes reading the README in raw markdown much easier and prevents super long lines.

```markdown
- [Create TTN account]
- [Add an application], take note of:
  - Application ID as `TTN_APP_ID`
- [Register two devices][Register your device]
  one for `ABP` and a second for `OTAA`
- Personalize one [device for ABP][Personalize device for ABP],
<!-- ... -->

[Create a TTN account]: (https://www.thethingsnetwork.org/docs/devices/node/quick-start/#create-an-account)
[Add an application]: https://www.thethingsnetwork.org/docs/applications/add/
[Register your device]: https://www.thethingsnetwork.org/docs/devices/node/quick-start/#register-your-device
[Personalize device for ABP]: https://www.thethingsnetwork.org/docs/devices/registration/#personalize-device-for-abp
```